object/get-dashboard.go

```diff
@@ -86,6 +104,50 @@ func GetDashboard(owner string) (*Dashboard, error) {
 			panic(err)
 		}
 	}()
+
+	go func() {
+		defer wg.Done()
+
+		if err := ormer.Engine.Find(&roles, &Role{Owner: owner}); err != nil {
+			panic(err)
+		}
+	}()
+
+	go func() {
+		defer wg.Done()
+
+		if err := ormer.Engine.Find(&groups, &Group{Owner: owner}); err != nil {
+			panic(err)
+		}
+	}()
+
+	go func() {
+		defer wg.Done()
+		if err := ormer.Engine.Find(&resources, &Resource{Owner: owner}); err != nil {
```

Comment:
The number of resources and transactions may be large, will it cause performance issue when it's number become large?

Reply:
Those numbers will not be very high in short term